test/integration/user_follows_place_test.rb

```diff
@@ -0,0 +1,40 @@
+require "test_helper"
+class UserFollowsPlaceSpec < ActionDispatch::IntegrationTest
+  include Capybara::DSL
+  require 'vcr'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.